geometry_msgs/PoseArray pose_array
float64 stiff1
float64 stiff2
